src/types/index.ts

```diff
@@ -490,3 +490,29 @@ export type AmountDetails = {
 export type Amount = {
   amount: number;
 };
+
+export type CollectData = {
```

Comment:
typo - this should match the iOS and Android types:
```suggestion
export type CollectedData = {
```